ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/formTemplates/shortDescription.html

```diff
@@ -0,0 +1,5 @@
+<template data-sly-template.shortDescription="${@ componentId, shortDescriptionVisible, shortDescription, bemBlock}">
+  <div id="${componentId}" class="${bemBlock}__shortdescription" data-sly-test="${shortDescriptionVisible && shortDescription}">
```

Comment:
If short description is not visible or enabled, you need to add its relevant BEM modifier